plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterRemovableChips.php

```diff
@@ -58,36 +52,35 @@ protected function render( $attributes, $content, $block ) {
 		<div <?php echo get_block_wrapper_attributes( $wrapper_attributes ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
 			<ul class="wc-block-product-filter-removable-chips__items">
 				<template
-					data-wp-each="state.items"
+					data-wp-each="state.activeFilters"
 					data-wp-each-key="context.item.uid"
 				>
 					<li class="wc-block-product-filter-removable-chips__item">
-						<span class="wc-block-product-filter-removable-chips__label" data-wp-text="context.item.label"></span>
+						<span class="wc-block-product-filter-removable-chips__label" data-wp-text="context.item.activeLabel"></span>
 						<button
 							type="button"
 							class="wc-block-product-filter-removable-chips__remove"
-							data-wp-bind--aria-label="context.item.removeLabel"
-							data-wp-on--click="<?php echo esc_attr( $parent_block . '::actions.removeFilter' ); ?>"
-							data-wp-bind--data-filter-item="context.item"
+							data-wp-bind--aria-label="state.removeActiveFilterLabel"
+							data-wp-on--click="actions.removeActiveFilter"
 						>
 							<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="25" height="25" class="wc-block-product-filter-removable-chips__remove-icon" aria-hidden="true" focusable="false"><path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></path></svg>
-							<span class="screen-reader-text" data-wp-text="context.item.removeLabel"></span>
+							<span class="screen-reader-text" data-wp-text="state.removeActiveFilterLabel"></span>
 						</button>
 					</li>
 				</template>
 				<?php foreach ( $filter_items as $item ) : ?>
 					<?php // translators: %s: item label. ?>
-					<?php $remove_label = sprintf( __( 'Remove filter: %s', 'woocommerce' ), $item['label'] ); ?>
+					<?php $remove_label = sprintf( __( 'Remove filter: %s', 'woocommerce' ), $item['activeLabel'] ); ?>
 					<li class="wc-block-product-filter-removable-chips__item" data-wp-each-child>
 						<span class="wc-block-product-filter-removable-chips__label">
-							<?php echo esc_html( $item['label'] ); ?>
+							<?php echo esc_html( $item['activeLabel'] ); ?>
 						</span>
 						<button
 							type="button"
 							class="wc-block-product-filter-removable-chips__remove"
 							aria-label="<?php echo esc_attr( $remove_label ); ?>"
-							data-wp-on--click="<?php echo esc_attr( $parent_block . '::actions.removeFilter' ); ?>"
-							data-filter-item="<?php echo esc_attr( wp_json_encode( $item, JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ) ); ?>"
+							data-wp-on--click="actions.removeActiveFilter"
+							data-wp-context="<?php echo esc_attr( wp_json_encode( array( 'item' => $item ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ) ); ?>"
```

Comment:
There is a helper for this case:

```suggestion
							<?php echo wp_interactivity_data_wp_context( array( 'item' => $item ) ); ?>
```

Reply:
Addressed in 9200fab932e1